src/libs/freetype/lv_freetype_glyph.c

```diff
@@ -136,8 +136,19 @@ static bool freetype_glyph_create_cb(lv_freetype_glyph_cache_data_t * data, void
     FT_Face face = dsc->cache_node->face;
     FT_UInt glyph_index = FT_Get_Char_Index(face, data->unicode);
 
-    FT_Set_Pixel_Sizes(face, 0, dsc->size);
-    error = FT_Load_Glyph(face, glyph_index,  FT_LOAD_COMPUTE_METRICS | FT_LOAD_NO_BITMAP | FT_LOAD_NO_AUTOHINT);
+    if(FT_IS_SCALABLE(face)) {
+        error = FT_Set_Pixel_Sizes(face, 0, dsc->size);
+    }
+    else {
+        LV_LOG_WARN("font is not scalable, selecting available size\n");
+        error = FT_Select_Size(face, 0);
+    }
+    if(error) {
+        FT_ERROR_MSG("FT_Set_Pixel_Sizes", error);
+        lv_mutex_unlock(&dsc->cache_node->face_lock);
+        return false;
+    }
+    error = FT_Load_Glyph(face, glyph_index,  FT_LOAD_COMPUTE_METRICS | FT_LOAD_NO_AUTOHINT);
```

Comment:
You can change the g_dsc->format = LV_FONT_GLYPH_FORMAT_IMAGE

Reply:
Thanks for the hint. So the format can be selected when loading the glyph in `lv_freetype_glyph`, but at that point, I am not sure if we know yet if it is a colored bitmap. 

TBH, I did not yet fully understand the flow. First the glyph is loaded:
https://github.com/lvgl/lvgl/blob/d7b82c79668d7b31d52f6535360eff15396eade5/src/libs/freetype/lv_freetype_glyph.c#L140
but then the glyph is loaded again but with different flags?
https://github.com/lvgl/lvgl/blob/d7b82c79668d7b31d52f6535360eff15396eade5/src/libs/freetype/lv_freetype_image.c#L131
Adding prints/or using a GDB shows that both paths are being hit when rendering a freetype font.